src/cljs/nr/utils.cljs

```diff
@@ -167,16 +167,26 @@
 (defn card-patterns-impl []
   "A sequence of card pattern pairs consisting of a regex, used to match a card
   name in text, and the span fragment that should replace it"
-  (letfn [(span-of [title code] [:span {:class "fake-link" :id code} title])
-          (regex-of [card-title] (re-pattern (regex-escape card-title)))]
+  (letfn [(span-of [title code] [:span {:class "fake-link" :id code} title])]
     (->> @all-cards
          (filter #(not (:replaced_by %)))
          (map (juxt :title :code))
-         (map (fn [[k v]] [(regex-of k) (span-of k v)]))
+         (map (fn [[k v]] [k (span-of k v)]))
          (sort-by (comp count str first) >))))
 
 (def card-patterns (memoize card-patterns-impl))
 
+(defn contains-card-pattern-impl []
+  "A card pattern regex, used to match a card name in text to check if the rest
```

Comment:
Not gonna make you change this before merging, but function docstrings have to go before the argument list `[]`, otherwise it's just a discarded string literal within the function.

Reply:
Thanks, sloppy by me, I just copied card-patterns-impl which does the same thing